src/Symfony/Component/Console/Command/DumpCompletionCommand.php

```diff
@@ -55,7 +55,7 @@ protected function configure()
         $this
             ->setHelp(<<<EOH
 The <info>%command.name%</> command dumps the shell completion script required
-to use shell autocompletion (currently, bash and fish completion is supported).
+to use shell autocompletion (currently, bash, zsh and fish completion are supported).
```

Comment:
maybe this list should be generated from `$this->getSupportedShells()`, to avoid having to remember this help text each time we add a new supported shell.

Reply:
Oups, I forgot this PR! Fixed!